quesma/quesma/search.go

```diff
@@ -813,30 +792,6 @@ func (q *QueryRunner) Close() {
 	logger.Info().Msg("queryRunner Stopped")
 }
 
-func (q *QueryRunner) findNonexistingProperties(query *model.Query, table *clickhouse.Table, queryTranslator IQueryTranslator) []string {
```

Comment:
We should check if we are referring to the non-existing columns before executing the query.  Our customer monitors the query log table. It can be moved to the last step of the transformation pipeline.

Reply:
the logic would have to be extended for the case of CTEs and subqueries (it didn't work properly there):

```
SELECT columnNotInSchema FROM (SELECT columnInSchema AS columnNotInSchema FROM table)
```

Not sure if I should address it in this PR.